common/common.go

```diff
@@ -135,3 +136,45 @@ func SupportsOverlay() bool {
 	}
 	return false
 }
+
+// PrivateNetList implements the flag.Value interface to allow specification
+// of -private-net with and without values
+type PrivateNetList struct {
+	mapping map[string]bool
```

Comment:
The `bool` values are used to check which index is contained and to check for existence by specifying the index. That doesn't work the the `struct{}`. See http://play.golang.org/p/VP2kPxie4P and compare the results.

Just be sure I'm not missing something obvious, how would you implement this with `struct{}`